src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -63,18 +108,71 @@ export class PoolListComponent {
     ];
   }
 
-  updateSelection(selection: CdTableSelection) {
-    this.selection = selection;
+  ngOnInit() {
+    this.summaryDataSubscription = this.summaryService.subscribe(
+      (data: any) => {
+        if (data) {
+          this.getPoolList(data.executing_tasks);
+        }
+      },
+      () => {
+        this.table.reset(); // Disable loading indicator.
+        this.viewCacheStatusList = [{ status: ViewCacheStatus.ValueException }];
+      }
+    );
   }
 
-  getPoolList(context: CdTableFetchDataContext) {
+  getPoolList(tasks: ExecutingTask[]) {
     this.poolService.getList().subscribe(
-      (pools: any[]) => {
-        this.pools = pools;
+      (pools: Pool[]) => {
+        this.pools = pools.map((pool) => {
+          pool.executingTasks = this.getPoolTasks(tasks, pool.pool_name);
```

Comment:
Have you reloaded the backend before trying? It works for me, I don't see a reason why it should not work.

Reply:
The pool creation works.

But, if it takes a long time, it will not show up in the table while it is being created.
If you go to RBD, the moment you finished using the creation form, it will show up in the table, even while it is still being created.

try adding a sleep of 15s in both backend methods and test it.

Reply:
The creation of a pool will happen nearly instantly, the other parts will take time like settings application metadata flags and of course the creation of PGs.

Reply:
I've add the safety net, as we cannot completely predict the behavior of a huge cluster. It can be easily replaced by your PR through the added tests. There were some issues with your PR that's why I could not include it here. I'll give you the feedback soon in your PR.